test/test_readability.py

```diff
@@ -91,27 +91,35 @@ def test_count_sentences_words_syllables():
 
 def test_flesch_score_range():
     for text in text_dict:
-        assert readability.Flesch().get_score_from_text(text) > 0
-        assert readability.Flesch().get_score_from_text(text) < 100
+        assert readability.FleschKincaidScorer().get_score_from_text(text) > 0
```

Comment:
Why do you need repeated instantiations of the FleschKincaidScorer class? Looks very inefficient.

Reply:
All the functions exposed are static, so this instantiation is not needed. Fixed